### DISCUSSION TITLE
PPOCRLABEL generate gt.txt for table recognition training dataset

### DISCUSSION THREAD

**Discussion by RickyGunawan09** [PaddlePaddle/PaddleOCR#12382] [Q&A]
hello thank you for making this beautiful repo. i have a question related PPOCRlabel.
i already annoted some table and when i see the result from auto generated gt.txt, i got some html tag its not right. especially for the merge cell.
i'm annoted the data like this:
![test_label](https://github.com/PaddlePaddle/PaddleOCR/assets/65578014/8d3d0a28-a3dc-468d-9236-76e8829dc5dc)

it is right ?

i got the result like this:
![result](https://github.com/PaddlePaddle/PaddleOCR/assets/65578014/beccf637-08c3-4432-9551-31fd15a5f059)
This is not similar to the table in the picture.

i expect the result should be like this:
![expected_result](https://github.com/PaddlePaddle/PaddleOCR/assets/65578014/8dc34b9d-5f56-4372-8e87-a271f63b5782)

Did I make a mistake during annotation? Or do you have to correct it manually even though you are already using PPOCRLABEL?

thank you

**Comment by tink2123** [comment]
The annotation of table recognition consists of two parts： box and html structures. 
The part mentioned above is the box label, which is very correct. 
In addition, the html annotation process needs to be completed in the pop-up excel. Please refer to the documentation for detailed steps3 : https://github.com/PaddlePaddle/PaddleOCR/tree/release/2.7/PPOCRLabel

```
Mark the table structure: for each cell contains the text, mark as any identifier (such as 1) in Excel, to ensure that the merged cell structure is same as the original picture.

Note: If there are blank cells in the table, you also need to mark them with a bounding box so that the total number of cells is the same as in the image.
```

**Comment by Bishow-99** [comment]
![Td](https://github.com/PaddlePaddle/PaddleOCR/assets/80660041/24865d1a-f825-4021-82d6-a854c04168ea)


@tink2123 Thank you in advance.  Can we train a model to recognize table structure(html format) in images like the ones above? If so, how do we annotate the data when a cell has multiple vertical alignments?
